client/post-editor/post-editor.jsx

```diff
@@ -695,19 +692,12 @@ export const PostEditor = React.createClass( {
 			this.props.resetEditorLastDraft();
 		}
 
-		// Assign editor post ID to saved value (especially important when
-		// transitioning from an unsaved post to a saved one)
-		if ( post.ID !== this.props.postId ) {
-			this.props.setEditorPostId( post.ID );
-		}
+		// Remove this when the editor is completely reduxified ( When using Redux actions for all post saving requests )
+		this.props.savePostSuccess( post.site_ID, this.props.postId, post, {} );
```

Comment:
Only by a bit of luck does this not cause a notice to be shown (which would be non-ideal, at least for now, given that it's redundant with the editor's own notice). See: https://github.com/Automattic/wp-calypso/blob/795cd4d6b863be46f567d689f81c0538a07c6074/client/state/notices/middleware.js#L78-L93 (it would show a notice if `{}` instead included a status property).

Arguably this is an argument against the lack of discoverability of middlewares 😄